src/lib/getApiError.js

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Missing /* ADD_LICENSE_HEADER */ from the first line

Reply:
Added